test/images/OWNERS

```diff
@@ -2,6 +2,7 @@
 
 reviewers:
   - mkumatag
+  - justaugustus
```

Comment:
@mkumatag @listx

Reply:
/lgtm
/approve

for k/test/images

/hold

for others to ack